CONTRIBUTING.md

```diff
@@ -98,11 +98,27 @@ If a flaky test cannot be fixed it should be removed from the suite.
 
 ## Testing new applications
 
-To test new applications you have to follow a similar process to
-[adding new tests](#adding-new-tests) however you will also need to configure
-this application to run the application through [docker compose][]
+To test new applications you will need to follow the
+[process for adding containers](docs/docker.md#adding-containers)
+to the [docker compose][].
 
-A brief guide to setting this up is available in
-[docs/docker.md](docs/docker.md#adding-containers).
+Once these are running inside of docker you should follow the
+[adding new tests](#adding-new-tests) process.
+
+When setting up the new RSpec tag such as `new_app: true` you will need to add
```

Comment:
I think this could benefit with some clues that you change new_app to your app name.

Suggestion for rephrasing this paragraph:

When adding a new app you should add a rSpec tag to associate tests with that app, and a step in the [Makefile](./Makefile) to run those tests. E.g. if you were adding whitehall as an app you would tag the tests with `whitehall: true` and create a step in the make file called `test-whitehall`.